src/content/blog/2024_qcoder.md

```diff
@@ -0,0 +1,163 @@
+---
+title: QCoder - A platform for quantum competitive programming
+author: Kein Yukiyoshi
+day: 20
+month: 10
+year: 2024
+tags: 
+  - quantum algorithms
+  - qiskit
+  - python
+---
+
+<p align="center">
+    <img src='../../images/2024-qcoder/qcoder_logo.png' alt='QCoder Logo' width=800/>
+</p>
+
+In recent years, quantum algorithms have attracted attention for their theoretical promise of significant efficiency gains.
+Shor's algorithm, for example, can factor integers with exponentially fewer queries than known classical algorithms.
+While learning about these algorithms is exciting, it can be challenging to stay motivated—especially for those without an academic background who are learning just for fun.
+Textbooks and academic papers are often too complex for beginners, making it hard to follow along.
+Even if you succeed in understanding and implementing an algorithm, running it on today’s quantum computers in scalable settings is typically impossible.
+
+In response to these challenges, we have developed <a href="https://www.qcoder.jp/" target="_blank">**QCoder**</a>, a platform for quantum competitive programming.
+QCoder is the quantum counterpart to conventional competitive programming platforms such as Codeforces, Google Code Jam, and ICPC.
+It allows users to learn quantum algorithms and practice quantum programming simply by participating in competitions.
+
+# Getting Started
+
+## Step 1: Environment Setup
+we currently support <a href="https://github.com/Qiskit/qiskit/" target="_blank">Qiskit</a> as the available quantum programming language.
+For the best coding experience, we recommend setting up a local Qiskit environment on your computer. This setup will provide useful features such as autocompletion and linting to improve productivity and code quality.
+
+To get started, please refer to the <a href="https://docs.quantum.ibm.com/guides/" target="_blank">official Qiskit documentation</a> for detailed installation instructions and guidelines.
+
+## Step 2: User Registratoin
+To participate in contests and submit your solutions, you need to sign up via the <a href="https://www.qcoder.jp/auth/signup" target="_blank">registration page</a>.
+
+## Step 3: Dive In and Enjoy!
+You now have access to all the content on QCoder. We suggest starting by participating in past contests. While you may find some parts difficult initially, you'll gain a deeper understanding as you solve problems and refer to the provided editorials. QCoder offers contests for everyone, from complete beginners to seasoned experts.
+
+# Sample Problem
```

Comment:
The sample problem idea is great! I think it should come before the `# Getting Started` section, however, so people can see what they're getting in to, and then sign up if they're interested. WDYT?

Reply:
I think that's a better way! I have modified the flow: https://github.com/unitaryfund/unitary.fund/pull/553/commits/3f0dbcd52a7f7cba82046d58bbf1b2b5ba0d72df